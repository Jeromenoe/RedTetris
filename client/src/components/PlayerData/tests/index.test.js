import React from 'react';
import { PlayerData } from '../index';
import renderer from 'react-test-renderer';
import configureStore from "redux-mock-store";
import {Provider} from 'react-redux';



describe('Testing PlayerData Component', () => {
	test('Test with basic values', async () => {
		const mockStore = configureStore();
		const initialState = {
			me: {
				level: 5,
				score: 1000,
				rows: 10
			}
		};
		const store = mockStore(initialState);
		const tree = renderer.create(<Provider store={store}><PlayerData /></Provider>).toJSON();
		expect(tree).toMatchSnapshot();
	  });
});