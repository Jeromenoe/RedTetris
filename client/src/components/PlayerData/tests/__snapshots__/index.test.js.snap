// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing PlayerData Component Test with basic values 1`] = `
<div
  style={
    Object {
      "alignSelf": "flex-end",
      "marginBottom": "6px",
      "marginRight": "6px",
      "width": "70px",
    }
  }
>
  <div
    className="sc-bdnxRM koyVdG"
  >
    <div
      style={
        Object {
          "marginBottom": "15px",
        }
      }
    >
      <div
        style={
          Object {
            "marginBottom": "6px",
          }
        }
      >
        Level
      </div>
      <span
        style={
          Object {
            "paddingRight": "6px",
          }
        }
      >
        5
      </span>
    </div>
    <div
      style={
        Object {
          "marginBottom": "15px",
        }
      }
    >
      <div
        style={
          Object {
            "marginBottom": "6px",
          }
        }
      >
        Score
      </div>
      <span
        style={
          Object {
            "paddingRight": "6px",
          }
        }
      >
        1000
      </span>
    </div>
    <div
      style={
        Object {
          "marginBottom": "20px",
        }
      }
    >
      <div
        style={
          Object {
            "marginBottom": "6px",
          }
        }
      >
        Lines
      </div>
      <span
        style={
          Object {
            "paddingRight": "6px",
          }
        }
      >
        10
      </span>
    </div>
  </div>
</div>
`;
